scale/job/tasks/base_task.py

```diff
@@ -281,7 +281,7 @@ def get_resources(self):
         """Returns the resources that are required/have been scheduled for this task
 
         :returns: The scheduled resources for this task
-        :rtype: :class:`job.resources.NodeResources`
+        :rtype: :class:`node.resources.node_resources.NodeResources`
         """
 
         raise NotImplementedError()
```

Comment:
I've started using `@abstractmethod` decorator instead of raise... I believe that is the proper way to do this now and future proof.